gene_expression_processing.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/env python3
+
+
+"""
+Hail Batch workflow to create gene expression files.
+This script will:
+
+- select genes to test based on expression (per cell type)
+- build chromosome & cell type specific phenotype covariate files
+- use gene info to create cis-window files
+- export pheno_cov files as tsv files
+- export cis_window files as tsv files
+
+More details in README
+output files in tob_wgs_genetics/saige_qtl/input
+"""
+
+import os
+import logging
+import math
+import sys
+
+import click
+import hail as hl
+import hailtop.batch as hb
+import pandas as pd
+import scanpy as sc
+
+from cpg_utils import to_path
+from cpg_utils.hail_batch import copy_common_env, dataset_path, output_path
+from cpg_workflows.batch import get_batch
+
+
+# use logging to print statements, display at info level
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s %(levelname)s %(module)s:%(lineno)d - %(message)s',
+    datefmt='%Y-%m-%d %H:%M:%S',
+    stream=sys.stderr,
+)
+
+CELLREGMAP_IMAGE = (
+    'australia-southeast1-docker.pkg.dev/cpg-common/images/cellregmap:0.0.1'
+)
+
+
+def filter_lowly_expressed_genes(expression_adata, min_pct=5) -> sc.AnnData:
```

Comment:
Do other similar scripts specify in whole percentages? I've seen this more commonly as a fraction (float from 0, 1) + it saves you a calculation.